ios-trivia-game/Utilities.swift

```diff
@@ -10,7 +10,28 @@ import Foundation
 import UIKit
 
 class Utilities {
-    static func quitGame(controller: UIViewController) {
+    static func quitGame(controller: UIViewController, roomId: String?, hasJoined: Bool = true) {
+        
+        if (roomId != nil && hasJoined) {
```

Comment:
by introducing hasJoined, we can account for @zhiachong 's code to decrement player count and removing games from the db only when you have truly joined the game, as opposed to quitting while creating the game.